src/omnisharp/server.ts

```diff
@@ -434,6 +439,9 @@ export class OmniSharpServer {
 
         let cleanupPromise: Promise<void>;
 
+        // Clear the session properties when the session ends.
+        this._sessionProperties = {};
```

Comment:
Ensure these properties are cleared between sessions. (i.e. Reload OmniSharp is invoked)